torch/csrc/distributed/rpc/request_callback_no_python.h

```diff
@@ -124,6 +124,12 @@ class TORCH_API RequestCallbackNoPython : public RequestCallback {
       RpcCommandBase& rpc,
       const c10::intrusive_ptr<JitFuture>& responseFuture) const;
 
+  // Helpers to run user-defined functions, operators and other computations.
+
+  c10::intrusive_ptr<JitFuture> runJitOperator(
```

Comment:
If this is not used in derived classes, this does not needs to be a member function?

Reply:
Later in the stack it will be used by `RequestCallbackImpl` (the subclass)